### ISSUE TITLE
the laucncher doest start

### ISSUE THREAD

**Issue by puseai** [remyroy/CDDA-Game-Launcher#601]
the launcher isnt starting whenever i press the launcher the command prompt thing only shows up for a second then disappears but if i run its as administrator it shows this "windows cannot find 'pythonw.exe' make sure you typed the name correctly and then try again" im on windows 10  32 bit pc i installed the latest version of cdda i tried both portable and installer both have same issue

**Comment by remyroy** [comment]
I don't think we support 32 bit. Have you tried it on a recent Windows 10 (64 bit) PC?
